api/prometheus/v1/api.go

```diff
@@ -67,6 +82,15 @@ const (
 	HealthGood    HealthStatus = "up"
 	HealthUnknown HealthStatus = "unknown"
 	HealthBad     HealthStatus = "down"
+
+	// Possible values for RuleType.
+	RuleTypeRecording RuleType = "recording"
+	RuleTypeAlerting  RuleType = "alerting"
+
+	// Possible values for RuleHealth.
+	RuleHealthGood    = "ok"
```

Comment:
Why not import them from here? https://github.com/prometheus/prometheus/blob/master/rules/manager.go#L46